src/components/Status/Status.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { useState } from 'react';
```

Comment:
Looks like `useState` can be removed from this file

Reply:
Removed